screwdriver/release-java-artifacts.sh

```diff
@@ -61,6 +61,7 @@ export FACTORY_VESPA_VERSION=$VESPA_RELEASE
 
 COMMON_MAVEN_OPTS="$VESPA_MAVEN_EXTRA_OPTS --no-snapshot-updates --settings $(pwd)/screwdriver/settings-publish.xml --activate-profiles ossrh-deploy-vespa -DskipTests"
 TMPFILE=$(mktemp)
+mvn $COMMON_MAVEN_OPTS  -pl :dependency-versions -DskipStagingRepositoryClose=true deploy 2>&1 | tee $TMPFILE
 mvn $COMMON_MAVEN_OPTS  -pl :container-dependency-versions -DskipStagingRepositoryClose=true deploy 2>&1 | tee $TMPFILE
```

Comment:
This line:
`mvn $COMMON_MAVEN_OPTS  -pl :container-dependency-versions -DskipStagingRepositoryClose=true deploy 2>&1 | tee $TMPFILE`
must be moved down below L70 and altered to:
`mvn $COMMON_MAVEN_OPTS  -pl :container-dependency-versions -DskipStagingRepositoryClose=true -DstagingRepositoryId=$STG_REPO deploy`

If not we are creating multiple staging repos and the dependency-versions will be in a separate orphaned staging repo.

Reply:
Thanks for spotting my copy/paste and providing a fix!